plasmapy/diagnostics/langmuir.py

```diff
@@ -302,7 +303,7 @@ def swept_probe_analysis(probe_characteristic, probe_area, gas,
     # The OML method is used to obtain an ion density without knowing the
     # electron temperature. This can then be used to obtain the ion current
     # and subsequently a better electron current fit.
-    n_i_OML, fit = get_ion_density_OML(probe_characteristic, probe_area, gas,
+    n_i_OML, fit = get_ion_density_OML(probe_characteristic, probe_area, gas.mass,
                                        return_fit=True)
```

Comment:
I got the recent tests to pass by removing `.mass` from this line - `get_ion_density_OML` uses the `Particle`'s attributes inside.